### ISSUE TITLE
Enter by itself not working

### ISSUE THREAD

**Issue by liamasis** [microsoft/vscode#179754]
Type: <b>Bug</b>

When i click enter it does not create a new line, but when i press command + enter it does.

VS Code version: Code 1.77.2 (e344f1f539a80912a0e9357cec841f36ce97a4e2, 2023-04-06T10:13:22.043Z)
OS version: Darwin arm64 21.6.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 3, 2|
|Memory (System)|8.00GB (0.11GB free)|
|Process Argv|--crash-reporter-id 80a93348-09f7-4b8a-8ab3-a05f9cfb0290|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-styled-components|die|0.1.33
es7-react-js-snippets|dsz|4.4.3
prettier-vscode|esb|9.10.4
auto-close-tag|for|0.5.14
auto-complete-tag|for|0.1.0
auto-rename-tag|for|0.1.10
vscode-docker|ms-|1.24.0
isort|ms-|2022.8.0
python|ms-|2023.6.0
vscode-pylance|ms-|2023.4.10
jupyter|ms-|2023.3.1000892223
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.288.0
vscode-jest|Ort|5.2.3
bracketeer|pus|1.3.1
vs-code-prettier-eslint|rve|5.1.0
console-ninja|Wal|0.0.105

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30610678
nodejswelcome1:30587005
3biah626:30602489
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
a9j8j154:30646983
vsccsb:30705552
pythonb192cf:30669361
defaultazurewalk:30687958
pythonms35:30701012
pythonfmttextcf:30710682

```

</details>

**Comment by ulugbekna** [comment]
Hi @liamasis ,

Thanks for filing the issue.

Could you please [try the following](https://github.com/microsoft/vscode/wiki/Keybinding-Issues#basic-troubleshooting):

- Run the command "Developer: Toggle Keyboard Shortcuts Troubleshooting" (you can, for example, run it from the Command Palette)
This will activate logging of dispatched keyboard shortcuts and will open an output panel with the corresponding log file.
- Press exactly the keybinding you desire.
- check what VS Code detects.

If you are having trouble understanding the output, please feel free to paste here and we'll be happy to help :-) 

---

Also, could you take a screen recording where the focus is when you're pressing `enter`? Please, before taking the recording, run command "Developer: Toggle Screencast Mode" so we see all key presses (the command can be run from the command palette - cmd+shift+p)

**Comment by addisonian** [comment]
Hey @liamasis,

I just went through this myself. The issue is coming from the vscode-styled-components extension. It inserted its own keybinding for the enter key. Even when I removed that keybinding, and changed the "command + enter" keybinding to just "enter", the enter key didn't behave how it used to.

I uninstalled the vscode-styled-components extension and restarted vscode and voila, the enter key is back to normal.
